docker/compose/demo/compose.yml

```diff
@@ -0,0 +1,128 @@
+version: "2.4"
+
+services:
+
+  dev_dataverse:
+    container_name: "dev_dataverse"
+    hostname: dataverse
+    image: gdcc/dataverse:alpha
+    restart: on-failure
+    user: payara
+    environment:
```

Comment:
In order to only have a single file to download, I copied the values out of .env into the compose file.